package/network/utils/iptables/Makefile

```diff
@@ -9,28 +9,26 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=iptables
-PKG_VERSION:=1.4.21
-PKG_RELEASE:=2
+PKG_VERSION:=1.6.0
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://www.netfilter.org/projects/iptables/files \
```

Comment:
Only http://www.netfilter.org/projects/iptables/files is a vaild source URL.
You can also add http://www.hu.netfilter.org and http://www.lt.netfilter.org as sources. 
Please place these before www.netfilter.org to keep main site offloaded.